src/main/java/com/hri/hri_web_backend/repository/AdministratorRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.hri.hri_web_backend.repository;
+
+import com.hri.hri_web_backend.domain.Administrator;
+import org.springframework.data.jpa.repository.JpaRepository;
+import java.util.Optional;
+
+public interface AdministratorRepository extends JpaRepository<Administrator, Long> {
+    Optional<Administrator> findByUsernameAndPassword(String username, String password);
```

Comment:
공백을 추가하면 가독성 향상에 도움이 될 것 같아요~

Reply:
혹시 어떻게 바꾸면 바꿀지 적어주실 수 있나요? 카멜케이스가 아닌가요?

Reply:
아 잘못 말씀드렸네요! 줄바꿈 넣으면 좋을 것 같습니다😅
```suggestion
    public interface AdministratorRepository extends JpaRepository<Administrator, Long> {
    
    Optional<Administrator> findByUsernameAndPassword(String username, String password);
```
이렇게요!